api/proto/info/info.proto

```diff
@@ -490,6 +515,37 @@ message ZInfoStorage {
   bool storageLocation = 4; // Storage location for app disks, images etc.
 }
 
+enum ZAppInstRunState {
```

Comment:
We don't want this duplication.

Reply:
These states are already defined in ZswState. Duplicating that serves no purpose and just leads to confusion and uncesary API transitions